common/main/en.xml

```diff
@@ -9246,5 +9248,26 @@ annotations.
 		<personName length="monogramNarrow" style="informal" order="givenFirst">
 			<namePattern>{given-informal-initial}</namePattern>
 		</personName>
+		<sampleName item="minimal">
+			<nameField type="given">Katherine</nameField>
+			<nameField type="surname">Johnson</nameField>
+		</sampleName>
+		<sampleName item="simple">
+			<nameField type="given">Alberto</nameField>
+			<nameField type="given2">Pedro</nameField>
+			<nameField type="surname">Calderón</nameField>
```

Comment:
For a Spanish name like this, would you want a `surname2` field as well?  Or would an example like that go somewhere else?

Reply:
We need to discuss in the meeting what kinds of example names we want for English.

Reply:
We don't have to have names for specific people. I think it is actually useful to have someone with surname2 for English, to show that we *don't* included in normal formatting there.

Reply:
Sounds good. I need to update the names here based on discussion in meeting; also need to add more same name item types. Will do that all in another PR.